src/httpServer/middleware.ts

```diff
@@ -84,3 +86,35 @@ export function KoaMiddleware(...koaMiddleware: Koa.Middleware[]) {
   }
   return clsdec;
 }
+
+/////////////////////////////////
+/* OPEN API DECORATORS */
+/////////////////////////////////
+
+// Note, OAuth2 is not supported yet.
```

Comment:
Maybe we don't need to exclude OAuth2 anymore since the decorator has no effect on Operon runtime. If developers want to integrate with OAuth2 in their app, they can use Koa middleware to do so (e.g., https://github.com/oauthjs/koa-oauth-server).

Reply:
Lets do this as a future PR. parsing the OAuth2 compile time info is much more complex than the other schemes, it requires a method level decorator to handle scopes and I'm not sure if/how OAuth scopes mixes with Operon roles.